params/json.libevm_test.go

```diff
@@ -127,20 +128,110 @@ func TestChainConfigJSONRoundTrip(t *testing.T) {
 			t.Cleanup(TestOnlyClearRegisteredExtras)
 			tt.register()
 
-			t.Run("json.Unmarshal()", func(t *testing.T) {
-				got := new(ChainConfig)
-				require.NoError(t, json.Unmarshal([]byte(tt.jsonInput), got))
-				require.Equal(t, tt.want, got)
-			})
+			expectedEncoded := bytes.NewBuffer(nil)
```

Comment:
Why is this change to the original test needed?

Reply:
I was just going over the existing test, and was wondering why we had subtests where it seems to me this makes it more convoluted? Maybe I missed something though. I can revert it of course, no worry.